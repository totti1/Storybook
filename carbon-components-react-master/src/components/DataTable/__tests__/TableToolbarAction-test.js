import React from 'react';
import { mount } from 'enzyme';
import { TableToolbarAction } from '../';

describe('DataTable.TableToolbarAction', () => {
  it('should render', () => {
    const wrapper = mount(
      <TableToolbarAction
        className="custom-class"
        iconName="add--glyph"
        iconDescription="Add"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
