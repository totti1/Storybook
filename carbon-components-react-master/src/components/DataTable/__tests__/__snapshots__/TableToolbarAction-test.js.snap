// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableToolbarAction should render 1`] = `
<TableToolbarAction
  className="custom-class"
  iconDescription="Add"
  iconName="add--glyph"
>
  <button
    className="custom-class bx--toolbar-action"
  >
    <Icon
      className="bx--toolbar-action__icon"
      description="Add"
      fillRule="evenodd"
      name="add--glyph"
      role="img"
    >
      <svg
        alt="Add"
        aria-label="Add"
        className="bx--toolbar-action__icon"
        fillRule="evenodd"
        height="16"
        name="add--glyph"
        role="img"
        viewBox="0 0 16 16"
        width="16"
      >
        <title>
          Add
        </title>
        <path
          d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm4 9H9v3H7V9H4V7h3V4h2v3h3v2z"
          key="key0"
        />
      </svg>
    </Icon>
  </button>
</TableToolbarAction>
`;
